CMakeLists.txt

```diff
@@ -63,6 +63,14 @@ if(BUILD_WITH_OPENMP)
     endif()
 endif()
 
+option(BUILD_EASYLOGGINGPP "Build EasyLogging++ as a part of the build" ON)
+
+if (BUILD_EASYLOGGINGPP)
```

Comment:
It should just be:
```
if (BUILD_EASYLOGGINGPP)
    add_definitions(-DBUILD_EASYLOGGINGPP)
endif()
```
This is like adding `BUILD_EASYLOGGINGPP` to the preprocessor definitions in a visual studio project

Reply:
Corrected.